src/common/procedure/src/local/rwlock.rs

```diff
@@ -18,6 +18,8 @@ use std::sync::{Arc, Mutex};
 
 use tokio::sync::{OwnedRwLockReadGuard, OwnedRwLockWriteGuard, RwLock};
 
+// Override warnings that those fields are not accessed.
+#[allow(dead_code)]
```

Comment:
Is here a `TODO` to drop this suppression?

Reply:
This doesn't look like a TODO. How would you like to proceed?

Reply:
Pushed b7a57b38f85953d07d4085cf1b3d43986a224e6e to resolve this.